--- make/config_.mk	2017-01-26 11:42:16.000000000 +0900
+++ make/config.mk	2017-01-24 15:40:16.000000000 +0900
@@ -162,7 +162,12 @@
 # You also need to add TORCH_PATH/install/lib to your LD_LIBRARY_PATH
 # TORCH_PATH = $(HOME)/torch
 # MXNET_PLUGINS += plugin/torch/torch.mk
+WSOFTMAX_PATH = $(HOME)/wsoftmax
+MXNET_PLUGINS += plugin/wsoftmax/wsoftmax.mk
  
+MULTIREGERSSION_PATH = $(HOME)/multiregression
+MXNET_PLUGINS += plugin/multiregression/multi_regression.mk
+
 # WARPCTC_PATH = $(HOME)/warp-ctc
 # MXNET_PLUGINS += plugin/warpctc/warpctc.mk
 
